nextflow.config

```diff
@@ -79,7 +80,15 @@ params {
     custom_config_base                  = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
 
 }
-
+// Validation for the min_contig_length parameter
+process {
```

Comment:
This is quite clever. However, we are using [nf-schema](https://github.com/nextflow-io/nf-schema) for parameter validation which means that the parameter type and constraints are defined in a schema file and the plugin automatically validates all the parameters. The schema file is here: https://github.com/Plant-Food-Research-Open/genepal/blob/dev/nextflow_schema.json

This schema can be automatically generated and refined through a web-based GUI. Please see the nf-core docs: https://nf-co.re/docs/nf-core-tools/pipelines/schema